README.md

```diff
@@ -0,0 +1,16 @@
+# bit-docs
+
+Loosely based on ideas and lessons learned from [DocumentJS](http://documentjs.com), bit-docs is an evolving set of tools that allow you to:
+
+ - Write documentation inline or in markdown files.
+ - Specify your code's behavior precisely with JSDoc
+   and [Google Closure Compiler](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler)
+   annotations.
+ - Generate multi-version documentation websites for your code from the inline and markdown documentation.
+ - Customize the theme and lay out the generated website.
+
+## Projects currently using bit-docs
+
+ - [CanJS](https://github.com/canjs/canjs)
+ - [StealJS](https://github.com/stealjs/stealjs)
+ - DoneJS (coming soon)
```

Comment:
This will rot over time unless there’s an issue for updating it; I would just link to the repo because we’ll probably merge `major` to `master` sometime in the next few weeks.